### ISSUE TITLE
NSL-KDD Features - Sheet1.csv

### ISSUE THREAD

**Issue by Malhabib** [Scorks/NetSecurity-ML#1]
I kindly request your assistance in obtaining the "NSL-KDD Features - Sheet1.csv" file. It would greatly facilitate my understanding and application of the code in the repository. If possible, could you please provide me with the mentioned file or guide me on where I can access it?

**Comment by Scorks** [comment]
You can get access to the required files here:

[https://www.unb.ca/cic/datasets/nsl.html](url)

**Comment by Malhabib** [comment]
hope this message finds you well. Firstly, I would like to express my appreciation for the work you have shared on your project repository. It has been a valuable resource for my own project and has provided me with valuable insights.

I have got this error when running the code, I tried to solve it but unfortunately couldn't make it.

TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp\ipykernel_18644\767063105.py in <module>
----> 1 generate_data(models_r, 'Generated_sample_rare.csv', train_new)

TypeError: generate_data() takes 2 positional arguments but 3 were given
